deployment/ansible/group_vars/development_template

```diff
@@ -9,7 +9,7 @@ otp_process_mem: 5G
 # for cac_secrets
 allowed_hosts: ['127.0.0.1', 'localhost']
 internal_ips: ['0.0.0.0', '127.0.0.1']
-postgis_version: [2, 1, 5]
+postgis_version: [2, 4, 4]
```

Comment:
The version of PostGIS on Amazon RDS for PostgreSQL is 2.1.8. We should try to match that in development as well (at least 2.1.x).

https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html#PostgreSQL.Concepts.General.FeatureSupport.Extensions.94x

It is possible that there isn't a specific package for 2.1.8—if that's the case, we've worked around it in the past by building the appropriate version of PostGIS from source:

https://github.com/azavea/cicero/blob/develop/deployment/ansible/roles/cicero.postgresql/tasks/install-postgis-from-source.yml